demo/kubernetes/README.md

```diff
@@ -1,150 +1 @@
-```
-
-Connect to the rook-client pod and verify the `rook` client can talk to the cluster:
-```
-kubectl -n rook exec rook-client -it bash
-rook node ls
-```
-
-At this point, you can use the `rook` tool along with some [simple steps to create and manage block, file and object storage](../client/README.md).
-
-### Teardown
-To clean up all the artifacts created by the demo, run the following:
-```
-kubectl delete deployment rook-operator wordpress wordpress-mysql
-kubectl delete thirdpartyresources cluster.rook.io
-kubectl delete namespace rook
-kubectl delete storageclass rook-block
-kubectl delete secret rook-rbd-user
-```
-If you modified the demo settings, additional cleanup is up to you for devices, host paths, etc.
-
-## Design
-
-With Rook running in the Kubernetes cluster, Kubernetes applications can
-mount block devices and filesystems managed by Rook, or can use the S3/Swift API for object storage. The Rook operator 
-automates configuration of the Ceph storage components and monitors the cluster to ensure the storage remains available
-and healthy. There is also a REST API service for configuring the Rook storage and a command line tool called `rook`.
-
-![Rook Architecture on Kubernetes](/Documentation/media/kubernetes.png)
-
-The Rook operator is a simple container containing the `rook-operator` binary that has all that is needed to bootstrap
-and monitor the storage cluster. The operator will start and monitor ceph monitor pods and a daemonset for the OSDs, which provides basic
-RADOS storage as well as a deployment for a RESTful API service. When requested through the api service,
-object storage (S3/Swift) is enabled by starting a deployment for RGW, while a shared file system is enabled with a deployment for MDS.
-
-The operator will monitor the storage daemons to ensure the cluster is healthy. Ceph mons will be started or failed over when necessary, and
-other adjustments are made as the cluster grows or shrinks.  The operator will also watch for desired state changes
-requested by the api service and apply the changes.
-
-The Rook daemons (Mons, OSDs, RGW, and MDS) are compiled to a single binary `rookd`, and included in a minimal container.
-`rookd` uses an embedded version of Ceph for storing all data -- there are no changes to the data path. 
-Rook does not attempt to maintain full fidelity with Ceph. Many of the Ceph concepts like placement groups and crush maps 
-are hidden so you don't have to worry about them. Instead Rook creates a much simplified UX for admins that is in terms 
-of physical resources, pools, volumes, filesystems, and buckets.
-
-Rook is implemented in golang. Ceph is implemented in C++ where the data path is highly optimized. We believe
-this combination offers the best of both worlds.
-
-See [Design](https://github.com/rook/rook/wiki/Design) wiki for more details.
\ No newline at end of file
+This file has moved to the [Documentation folder](/Documentation/kubernetes.md)
```

Comment:
I see.  Yes, I think that "this file has moved" experience is OK for random markdown files, but not for the default `README.md` because github automatically renders it whenever the folder is browsed to, so it should have some useful intro/guidance.